scripts/create_aggregated_zone_config.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+"""
+Create an aggregation zone config file by looking up sub zones based on the zone name.
+Sums capacities, all contributors.
+poetry run python scripts/create_aggregated_zone_config US US-Central
+"""
+import argparse
+from copy import deepcopy
+from pathlib import Path
+
+import yaml
+
+from electricitymap.contrib.config.model import Capacity, Zone
+
+NAME_MAPPING = {"hydro storage": "hydro_storage", "battery storage": "battery_storage"}
+
+
+def create_aggregated_config(zoneKey: str, timezone: str):
+    zone = Zone(
+        key=zoneKey,
+        sub_zone_names=[],
+        timezone=timezone,
+        contributors=[],
+        capacity=Capacity(),
+    )
+    zone.parsers = None
+    for path in Path("config/zones").glob(f"{zoneKey}-*.yaml"):
+        if path.is_file():
+            with open(path, "r") as file:
+                subzone = yaml.safe_load(file.read())
+                zone.sub_zone_names.append(path.stem)
```

Comment:
```suggestion
if instanceof(zone.sub_zone_names, list):
    zone.sub_zone_names.append(path.stem)
```
Improves type saftey as yaml.safe_load can return None or a Dict.

However it looks like it's used as a dict on the next line?